packages/massa-web3/src/experimental/account.ts

```diff
@@ -224,26 +226,33 @@ export class Account {
   static async fromEnv(): Promise<Account> {
     return Account.fromPrivateKey(PrivateKey.fromEnv())
   }
-}
 
-type AccountKeyStore = AccountV0KeyStore | AccountV1KeyStore
+  /**
+   * Uses the environment variables to create an account.
+   *
+   * @returns An account instance.
+   */
+  static async fromYaml(path: string, password: string): Promise<Account> {
+    // check that file exists
+    if (!existsSync(path)) {
+      throw new Error(`Wallet file "${path}" does not exist.`)
```

Comment:
In a similar way to what we are doing with Go, I propose starting exception messages with a lowercase letter, as they should be composed. Except if that's not idiomatic in TS.